src/menu_events.cpp

```diff
@@ -1489,8 +1489,8 @@ void console_handler::do_droid()
 		const bool is_droid = menu_handler_.board().get_team(side).is_droid();
 		const bool is_proxy_human = menu_handler_.board().get_team(side).is_proxy_human();
 		const bool is_ai = menu_handler_.board().get_team(side).is_ai();
-		const std::string human = team::CONTROLLER::enum_to_string(team::CONTROLLER::HUMAN);
-		const std::string ai = team::CONTROLLER::enum_to_string(team::CONTROLLER::AI);
+		const std::string human = side_controller::human;
+		const std::string ai = side_controller::ai;
```

Comment:
The original code seems to be making local aliases to avoid the long enum_to_string code. Now that the right-hand-side is just side_controller::human, I don't think the alias is needed.

Reply:
Fixed.